.flake8

```diff
@@ -0,0 +1,54 @@
+[flake8]
+
+builtins = _
+
+# Print the total number of errors:
+count = true
+
+# Don't even try to analyze these:
+extend-exclude =
```

Comment:
Should this also include `.tox` so it doesn't try to flake8 the entire test folder structure?

Reply:
yup, good catch, TY